client/actions/loginActions.js

```diff
@@ -31,9 +27,7 @@ export function signupUser(body) {
             .then((response) => {
                 dispatch(signupAction(response.data))
             })
-            .catch((err) => {
-                console.log("error =====>>>>", err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}